spec/forms/support_interface/application_forms/edit_reference_details_form_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::ApplicationForms::EditReferenceDetailsForm do
+  let(:reference) { create(:reference, name: 'John Doe', email_address: 'john.doe@example.com', relationship: 'Colleague') }
+
+  describe 'validations' do
+    subject { described_class.new }
+
+    it { is_expected.to validate_presence_of(:name) }
```

Comment:
You are using shoulda matchers here 😆 

https://github.com/DFE-Digital/publish-teacher-training/pull/4428#discussion_r1710739143

Reply:
Yes, I was reluctant to do so ... Following Apply patterns but I wish this gem was off this app.